.eslintrc.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  env: {
+    es2021: true,
+    node: true,
+    mocha: true
+  },
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 12,
+    sourceType: 'module'
+  },
+  plugins: ['@typescript-eslint', 'prettierx'],
+  extends: [
+    'standard',
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
```

Comment:
I do not remember that one contains `@typescript-eslint/recommended-requiring-type-checking ` ...

Reply:
Seems like setting `plugin:@typescript-eslint/recommended-requiring-type-checking` in the current stage with a `tsconfi.lint.json` produces to much errors right now, because also tests and other js files are affected by this

I would vote for creating another branch for that later